src/Test.hs

```diff
@@ -12,37 +12,38 @@ inheritMyStyle = inheritStyleHeaderGroup makeLineSolid id (fst . style) (snd . s
     style TinySep  = (SingleLine, NoLine)
 
 main :: IO ()
-main = putStrLn $ tableString [ column (expandUntil 30) left (charAlign ':') ellipsisCutMark
-                              , column expand center noAlign noCutMark
-                              ]
-                              unicodeRoundS
-                              noneH
-                              (titlesH ["Layout", "Result"])
-                              rowGroups
+main = putStrLn $ tableString $ horizontalHeaderTableS
```

Comment:
Where is `horizontalHeaderTableS` defined? This doesn't seem to compile for me.

Reply:
I changed the name and forgot this instance.